add relative positioning to html, body; add fixed positioning to .attention, for Firefox compatibility

ItsNickBarry opened the pull request:
Firefox was displacing part of the "?" button and displaying the contact link too early.  Adding `position: fixed;` to `.attention` fixed the "?" button.  Then, adding `position: relative` to `body` actually fixed both issues.  I left the first fix because there's a joke in there somewhere.

Here's what was hapening:
![screenshot from 2015-07-08 01 09 03](https://cloud.githubusercontent.com/assets/1500628/8566132/b8797528-2510-11e5-8a6c-723971b0df6e.png)

ItsNickBarry commented:
Also, if David Runger were with us today, he would advise you to delete the master branch, though the debate is ongoing.

jacobhamblin commented:
Good catch! Thanks for the help. How do you personally feel about master + gh-pages?

Out of (ignorance) fear of messing something up, I've kept master, but if master is the same as any other branch, having just gh-pages would be easier for me to maintain.

ItsNickBarry commented:
I'm glad to do it.  My goal is to fix a Firefox compatibility issue for everyone I met at App Academy.

I have removed master from my hyperbolic-canvas without issue.  The name is just a convention, I think.  However, I did some testing today with `gh-pages` repositories vs the _username_.github.io repository, and there were some issues.

As you can probably tell, there is potential for name collisions between `gh-pages` repositories, and _username_.github.io repositories.  I wanted to see which would take precedence, so I added some test files to my _username_.github.io, which matched those in my hyperbolic-canvas repository.  At first, the `gh-pages` branch of hyperbolic-canvas took precedence.  However, once I deleted the test files from my _username_.github.io, they started showing up instead.  I assumed that it was some sort of server-side caching delay, but it went on for about 10 minutes.  The problem was fixed when I deleted and replaced my _username_.github.io.

Maybe the cache would have updated, given enough time, and served files from one of the repositories consistently.  The URL collision would still exclude the other, however.

One potential solution to this is to get rid of the `gh-pages` branch from all of your repositories, and instead include them in your _username_.github.io as submodules.  This basically puts a repository into another repository.  The disadvantages are that your _username_.github.io might get too large over time, given enough submodules, and that it might have to be updated manually, which is the problem you're having in the first place.  I'm not entirely sure about this, though, as I learned about submodules only today.

You can sort of see how submodules work on my [gmio](https://github.com/ItsNickBarry/gmio) repository, part of which is now in the [neat](https://github.com/ItsNickBarry/neat) repository.  Read about them [here](https://www.git-scm.com/book/en/v2/Git-Tools-Submodules).
